docs/assets/diagrams/transfer/03a-confirm-payee.puml

```diff
@@ -0,0 +1,127 @@
+@startuml
+
+title Confirm Payee
+
+hide footbox
+
+autonumber
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  database Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Third-party\nAPI Adapter" as Adapter
+end box
+
+Alice -> App: The payee info is correct
+
+rnote right of App #Light
+  ""updateDoc(""
+  ""  "transactions/users/1234/<uuid>",""
+  ""  {""
+  ""    sourceAccountId: "bob.fspa",""
```

Comment:
I think it's up to the app to decide (cc @MahidharBandaru). However, the app needs to pass in the value at this step because this will be the one that triggers calling `POST thirdpartyRequests/transactions`.

Reply:
Currently the user chooses where the money is coming from first.